src/common/models/Projection.ts

```diff
@@ -299,13 +284,15 @@ export class Projection {
       summaryWithTimeseries.actuals.cumulativeDeaths;
     this.currentCumulativeCases =
       summaryWithTimeseries.actuals.cumulativeConfirmedCases;
-    this.currentContactTracerMetric = lastValue(this.contractTracers);
+    this.currentContactTracerMetric = metrics
+      ? metrics!.contactTracerCapacityRatio
+      : null;
   }
 
   get currentContactTracers() {
     return (
       CONTACT_TRACER_STATE_OVERRIDES[this.stateName] ||
-      lastValue(this.actualTimeseries.map(row => row && row.contactTracers)) ||
+      this.metrics?.contactTracerCapacityRatio ||
```

Comment:
nit: This would coerce `0` to null, but since there are no states with 0 contact tracers, it's probably fine. 😁

Reply:
I actually think this is wrong anyways, this should be the raw number of contact tracers, not the ratio